webpack.config.js

```diff
@@ -0,0 +1,21 @@
+// eslint-disable-next-line import/no-unresolved
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const path = require('path');
+
+const ruleForStyle = {
+  test: /\.css$/,
+  use: ['style-loader', 'css-loader'],
+};
+
+const rules = [ruleForStyle];
+
+module.exports = {
+  entry: './src/index.js',
+  output: {
+    path: path.resolve(__dirname, 'build'),
+  },
+  plugins: [
+    new HtmlWebpackPlugin({ template: 'src/index.html' }),
+  ],
+  module: { rules },
+};
```

Comment:
- [ ] Kindly set up the webserver, as shown on the exercise before
![image](https://user-images.githubusercontent.com/73140461/149181678-95c3e268-9ea6-40c1-b3f1-92f5bd2aa764.png)

Reply:
I honestly do not understand what I have to configure, since you can clearly see how it is configured in the exercise

Reply:
all fixed and set up the webserver